pkg/api/validate/dynamic/dynamic.go

```diff
@@ -554,8 +557,12 @@ func (dv *dynamic) validateVnetLocation(ctx context.Context, vnetr azure.Resourc
 	return nil
 }
 
+var (
```

Comment:
it's more conventional to declare vars before any functions in the beginning of the file

Reply:
updated